src/openrct2/util/Util.cpp

```diff
@@ -21,6 +21,9 @@
 #include <cctype>
 #include <cmath>
 #include <ctime>
+#include <random>
+
+static std::mt19937 _prng;
```

Comment:
Is this thread safe?

Reply:
Are we calling this from across threads currently? rand() isn't either or at least its not guaranteed and implementation specific.

Reply:
I've changed it to use thread_local and is seeded with a number from random_device during initialization, this should do it and eliminates the need to call util_srand before usage.